app/models/patient.rb

```diff
@@ -6,7 +6,15 @@ class Patient < ApplicationRecord
 
   validates :name, presence: true
 
+  before_save :remove_name_whitespace
+
   def deletable?
     event_procedures.empty?
   end
+
```

Comment:
Maybe you can solve it like this also 🤔, with this way you will kinda override the `name setter`. 
Makes sense?
```suggestion
def name=(value)
  self.name = value.strip
end
```

Reply:
Totally, I followed your suggestion and change the setter using `super`. Done!